exercises/practice/dot-dsl/src/main/java/DotDsl.java

```diff
@@ -0,0 +1,65 @@
+import java.util.Collection;
+import java.util.Map;
+
+class DotDsl {
+
```

Comment:
I will probably also tend to give the `Element` class structure to the user instead of them having it to implement it from scratch